sdk/communication/communication-messages-rest/samples-dev/Notification_SendTemplateMessage.ts

```diff
@@ -65,8 +64,8 @@ async function main() {
     const  result = await client.path("/messages/notifications:send").post({
         contentType: "application/json",
         body: {
-            channelRegistrationId: "<CHANNEl_ID>",
-            to: ["<PHONE_NUMBER>"],
+            channelRegistrationId: "972e282c-ea60-4bc7-bd22-b9e0b3d99bd3",
+            to: ["+14014507114"],
```

Comment:
Is this intended?

Reply:
I have fixed it now. It will fetch the values from environment variable.